assets/app/features/upgrade/controllers/upgrade-backup.controller.spec.js

```diff
@@ -210,7 +186,9 @@ describe('Upgrade Flow - Backup Controller', function() {
                     });
 
                     it('does not use the default name for the file', function () {
-                        expect(downloadBackupMock.download).not.toEqual('crowbarBackup');
+                        expect(FileSaver.saveAs).not.toHaveBeenCalledWith(
+                          mockedDownloadResponse.data, 'crowbarBackup'
```

Comment:
You can use this place to do some basic blackbox testing of `extractFilename()`, i.e. expect the actual filename from headers here.